src/specs/models/plots-tables-schemas/PlotData.v1.yaml

```diff
@@ -0,0 +1,8 @@
+title: Plot Data
+type: object
+properties:
+  plotData:
+    type: array
+required:
+  - plotData
```

Comment:
what about the plotDataKey parameter?

Reply:
This validation is only on the plotData that is returned from the new S3 bucket, which is just the list of values used in the plots/

Reply:
I see. I didn't know that we do validation also on what is returned by AWS. What about validating what is returned from the pipelines repo? Do we do validation on that?